bin/jetify

```diff
@@ -13,8 +13,20 @@ do
 done <<< "$(cat $MAPPING_FILE)"
 echo "Command file for sed ready. Searching for files to jetify..."
 
-for FILE in `find $PROJECT_DIR \( -name "*.java" -o -name "*.xml" \) -type f -not -path '*/\.git*' | sort`; do
-  echo "jetifying file $FILE"
+jetify() {
+  echo "jetifying file $1"
   sed -i".bak" -f $MAPPING_SED $FILE
   rm -f "$FILE.bak"
+}
+
+N=128
+
+find . -type f -name "*_peaks.bed" ! -path "./tmp/*" ! -path "./scripts/*" 
+for FILE in `find $PROJECT_DIR \( -name "*.java" -o -name "*.xml" \) -type f -not -path '*/\.git*' ! -path "*/android/app/build/*" ! -path "*/android/build/*"`; do
+  (
+    jetify $FILE
+  ) &
+  if [[ $(jobs -r -p | wc -l) -gt $N ]]; then
```

Comment:
this is pretty sweet! apart from the maybe-problem with android/build false-positives this could be huge

Reply:
Yeah, totally agreed. I think there's a faster way we can do this faster without skipping over these files, I'll get back to you about it shortly!

Reply:
that speed was incredible though, I just tested it. My only concern now is that I have a very burly machine compared to many I personally know and it was pegged - should probably make the parallelism configurable with a command line arg if this is your final design
(it would work for me this way, btw - just want people to be able to tune it down if they need). Really cool